### PR TITLE
Terraform ecs task

### PR DESCRIPTION
**Pull request by Kurt812**
Added an ECS task definition to the terraform file, also added some policies for the task so that it was able to add files to the S3 bucket. 

Have ran terraform apply and then ran the task manually, all appears successful with a new paraquet file being uploaded to the output S3 bucket.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -66,3 +66,97 @@ resource "aws_lambda_function" "c14-team-growth-lambda" {
   }
 }
 
+data "aws_iam_role" "ecs_task_execution_role" {
```
Comment (S1mpySloth) [inline terraform/main.tf:69]: Perhaps consider making the security group within terraform rather than on AWS alone?

File: terraform/main.tf [hunk 2]
```diff
@@ -66,3 +66,97 @@ resource "aws_lambda_function" "c14-team-growth-lambda" {
...
+        },
+        Action    : "sts:AssumeRole"
+      }
+    ]
+  })
+}
+
+resource "aws_iam_policy" "ecs_task_s3_policy" {
+  name        = "c14-team-growth-ecs-task-s3-policy"
+  description = "Allow ECS tasks to upload files to the S3 bucket"
```
Comment (S1mpySloth) [inline terraform/main.tf:96]: What kind of files?

File: terraform/main.tf (hunk 1 above)
Comment (Kurt812) [reply terraform/main.tf:69]: Have added a security group for when we run the tasks, will need to be attached to the scheduler.

**Comment by S1mpySloth** [comment]
Happy with changes, cleared to merge :)

**Pull request merged**